src/components/List/tokenList.js

```diff
@@ -8,9 +8,10 @@ export const AddToken = () => {
   const handleNewToken = () => {
     if (!tokenLocal) {
       const newToken = getToken();
+      setToken(newToken);
       localStorage.setItem('token', newToken);
     }
-    window.location.href = '/list';
+    window.location.href = 'http://localhost:3000/list-shopping';
```

Comment:
🔴 This should be `/list` - you can see my comment in `/home.js` for more context.

Reply:
Listo ! :+1: